src/theme/css/variables.css

```
@@ -60,4 +60,4 @@
         
         --chapter-hover-bg: var(--theme-hover);
     }
-}
+}
```

Do end files with a blank line
```suggestion
}

```

---

Ok, next time